ppx_lambda_vm/ppx_lambda_vm.ml

```diff
@@ -143,6 +142,14 @@ let rec expr_of_ocaml_expr expr =
     let first = expr_of_ocaml_expr first in
     let second = expr_of_ocaml_expr second in
     [%expr Pair { first = [%e first]; second = [%e second] }]
+  | Pexp_let
```

Comment:
I will approve, but leave this comment here
```suggestion
  (* TODO: should let be removed? *)
  | Pexp_let
```

Reply:
All  right, as soon as we have something to compile from Ligo I'll do that.

Reply:
Done!